packages/renderer/src/routes.ts

```diff
@@ -0,0 +1,21 @@
+export class PodDetailsRoute {
```

Comment:
Hello, I think we should not add code in src/ folder, as it's dedicated to Pods probably it can go as new method of the pod-utils


https://github.com/containers/podman-desktop/blob/main/packages/renderer/src/lib/pod/pod-utils.ts

Reply:
My idea was to add new class/methods for each route defined in `App.svelte` (at least the ones with parameters), which are not all related to pods, and have these methods at the same level of `App.svelte`

Reply:
I would say we need to split "code refactoring" and "fixes"

so do not try to change all routes of App.svelte and fix the encoding at the same time.

We need separate iteration/steps
one PR = one goal (can be refactoring (no-op) or bug fixes)

I don't see value in just adding a dedicated object just for routes as routes are specific to some area (like pods, containers, images, etc and we already have utility there)

in PodList we already have the object there